daisy_workflows/image_build/debian/debian_11_worker_arm64.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
```

Comment:
i don't think we need a completely cloned file for this; can we just introduce some 'if' branching to the main script instead?